### PR TITLE
feat(my-joyent): Adds boilerplate for my joyent

### PR DESCRIPTION
**Pull request by SaraVieira**
Create starting boilerplate for my joyent prototype

This also fixes precommit script

### CODE REVIEW

File: packages/my-joyent/scripts/patch-webpack-config.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
+const webpack = require('webpack');
```
Comment (ramitos) [inline packages/my-joyent/scripts/patch-webpack-config.js]: We should consider detaching this into it's own package.

File: yarn.lock [hunk 2]
```diff
@@ -2,73 +2,17 @@
 # yarn lockfile v1
```
Comment (ramitos) [inline yarn.lock]: How are all this packages being removed? :|

can you run: 
```
rm -rf node_modules; rm -rf packages/*/node_modules; rm packages/*/package-lock.json; rm packages/*/yarn.lock; rm -rf packages/*/dist; rm -rf ~/.yarn-cache; npm cache clean --force; yarn cache clean; rm yarn.lock; yarn install
```

File: packages/my-joyent/scripts/patch-webpack-config.js (hunk 1 above)
Comment (SaraVieira) [reply packages/my-joyent/scripts/patch-webpack-config.js]: Issue created : https://github.com/yldio/joyent-portal/issues/640

File: yarn.lock (hunk 2 above)
Comment (SaraVieira) [reply yarn.lock]: Can you check ? I think it's fixed

**Comment by SaraVieira** [comment]
🎉

**Pull request merged**